cmd/plakar/utils/utils.go

```diff
@@ -307,3 +308,15 @@ func NormalizePath(path string) (string, error) {
 
 	return normalizedPath, nil
 }
+
+var ansiEscapeSeq = regexp.MustCompile(`\x1b\[[0-9;?]*[ -/]*[@-~]`)
```

Comment:
the regex is both:
- too specific : match only some of `ESC` escapes codes
- not enough specific : doesn't match some others control characters (like `BELL` or `BS` aka backspace)

it might be better to just escape all C0 controls chars (0x00 to 0x1f). `ESC` is 0x1b, so it is in the range.

see [ECMA-48](https://ecma-international.org/publications-and-standards/standards/ecma-48/), section 5.2.

Reply:
and also `CSI` should be escaped as it introduces Control Sequence, see 5.4 (it is 0x9b, if I assume only 8-bits   character set to be supported).